### ISSUE TITLE
Update support-directory.md

### ISSUE THREAD

**Issue by gratefulcheddar** [decred/dcrdocs#341]
Update https://docs.decred.org/support-directory/ with new contractors if they'd like to be listed.

**Comment by jholdstock** [comment]
**Remove:**
dbt
coin artist
grateful cheddar

**Add (with their approval):**
peter zen - copay dev
go1dfish - decrediton dev
sndurkin - decrediton dev
ingsoc - community management
haon - community management
pvtwarren  - community management

**Update:**
Marco - no longer IRC only

**Comment by jholdstock** [comment]
Linutee requested changes to designer section:
```
@karamble: Web Designer 
Website Management
General Support

@kart: Illustrator/Designer
Illustrations
Graphic Design

@kyle_firethought: Motion Designer
Animation and Motion Design
Interactive Media
Video Creation

@linnutee: Lead Designer
Identity
UI/UX Design
Visual Design
Motion Design

@lustosa: Illustrator
Illustrations
Graphic Design

@pflan: Designer
UI/UX Design
Front-End Development
Visual Design

@sander: Designer 
Identity
Visual Design
UI/UX Design
Front-End Development
```

**Comment by underscoreingsoc** [comment]
Updated "Community Support" section:

```
@bacon
Discord

@dyrk
Bitcointalk
Slack
Russia

@emiliomann
Bitcointalk
Facebook
Slack
Brazil

@haon
Facebook
Slack
Netherlands

@ingsoc
Discord
Slack
Twitter

@jd
Facebook
YouTube

@jz
Reddit
Slack

@praxis
Bitcointalk
Reddit

@pvtwarren
Reddit
Slack

@thedecreddigest
Twitter
```

New "Marketing Support" section:

```
@bacon
Media

@haon
Media

@iizuka
Advertising

@ingsoc
Networking

@jd
Advertising

@jiecut
Media

@jjb
Advertising

@joshuam
Networking

@jz
Networking

@solar
Advertising

@thedecreddigest
Media

@timhebel
Advertising
```

We should think about restructuring the entire Support Directory given how many contributors are now participating. The new team page being developed for the website should provide insight. Another option is using icons to indicate simplified areas of activity/expertise to avoid duplication and overlap.

**Comment by jholdstock** [comment]
I've discussed with @underscoreingsoc and we think it is sensible to limit the listed contractors on the page to only include contractors who have signed contracts. Some of the names listed above are still in the "vetting" stage, so we will leave those out for now.

**Comment by jholdstock** [comment]
This was meant to be closed when #360 was merged. Can somebody close it please?

**Comment by jolan** [comment]
Closed via #360.
